blivet/formats/disklabel.py

```diff
@@ -324,6 +324,15 @@ def logicalPartitions(self):
         return logicals
 
     @property
+    def primaryPartitions(self):
+        try:
+            primaries = self.partedDisk.getPrimaryPartitions()
+        except Exception: # pylint: disable=broad-except
```

Comment:
He was just following the style of the existing, related properties. We should probably drop the try/except on all three of them, but I have some concerns about doing that in a minor version release.